Makefile

```diff
@@ -23,7 +23,6 @@ karate:
 lint:
 	npm run lint-oas
 	npm run lint-karate-js
-	npm run lint-node-sandbox
```

Comment:
This should not be required by the end of this work, as there should only be the Dockerfile in the `sandbox` directory - ie there will be nothing to lint.